dev/breeze/src/airflow_breeze/commands/developer_commands.py

```
@@ -625,6 +631,7 @@ def start_airflow(
         force_build=force_build,
         forward_credentials=forward_credentials,
         github_repository=github_repository,
+        go_worker=go_worker,
```

We should probably add one check though, we should validate if the executor is `CeleryExecutor` or fail early